app/utils/data.test.ts

```
@@ -0,0 +1,361 @@
+
+describe('omit()', () => {
+  it('should work correctly', () => {
+    expect(omit({ b: 1, a: 2 }, ['a'])).toStrictEqual({ b: 1 });
+  });
+
+  // TODO more tests for edge cases
+});
+
+describe('uniq helpers', () => {
+  let origArr: any[], uniqArr: any[];
+  beforeEach(() => {
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)